src/App.tsx

```diff
@@ -75,7 +73,12 @@ const App: React.FC = () => {
                 <Route index element={<HomePage products={products} />} />
 
                 <Route path="phones">
-                  <Route index element={<PhonesPage phones={phones} />} />
+                  <Route
```

Comment:
Great you did it with your product page!
Let's improve a bit more)
let's make our paths to product pages as variables for instance `:category` and make nested route. props will be passed depending on url